homeassistant/components/mqtt/cover.py

```diff
@@ -217,6 +218,8 @@ def __init__(self, hass, config, config_entry, discovery_data):
         self._optimistic = None
         self._tilt_value = None
         self._tilt_optimistic = None
+        self._has_tilt_topic = None
```

Comment:
Let's not add an instance variable for something that is only used once.

Reply:
Agree